__tests__/app.test.js

```diff
@@ -14,21 +14,26 @@ describe("/api/*", () => {
       .get("/api/fjdihdkfkhudz")
       .expect(404)
       .then((response) => {
-        expect(response.body.msg).toBe("invalid path");
+        expect(response.body.msg).toBe("Invalid path");
       });
   });
 });
 
 describe("/api", () => {
   test("GET 200: responds with a description of all other endpoints available", () => {
-    return readEndpoints().then((result) => {
-      return request(app)
-        .get("/api")
-        .expect(200)
-        .then((response) => {
-          expect(response.body.endpoints).toEqual(result);
-        });
-    });
+    return fs
+      .readFile(
+        "/Users/tiahhind/Northcoders/backend/be-nc-news/endpoints.json",
+        "utf8"
```

Comment:
Using readFile is fine, if you want you can also just require in the endpoints file content with a const / require